src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -309,8 +309,12 @@ function MoneyRequestView({report, shouldShowAnimatedBackground, readonly = fals
 
             // Return violations if there are any
             if (hasViolations(field, data, policyHasDependentTags, tagValue)) {
-                const violations = getViolationsForField(field, data, policyHasDependentTags, tagValue);
-                return ViolationsUtils.getViolationTranslation(violations[0], translate);
+                const violations = getViolationsForField(field, data, policyHasDependentTags, tagValue) as OnyxTypes.TransactionViolation[];
```

Comment:
Please, make sure `getViolationsForField` returns `OnyxTypes.TransactionViolation[]` same way as it was before.
```suggestion
                const violations = getViolationsForField(field, data, policyHasDependentTags, tagValue);
```